.sync.yml

```diff
@@ -8,7 +8,16 @@ Gemfile:
         platforms: ["mswin", "mingw", "x64_mingw"]
         version: '< 2.6.0'
 .travis.yml:
-  unmanaged: true
```

Comment:
So this file's now unmanaged but we're adding settings for it?

Reply:
Correct.  We are now managing this file with the specified settings.